### PR TITLE
Add `LinearLR` compute lr formula in doc

### LINKED ISSUE #68058: Some lr-schedulers docs seems to have typos/missing information
## 📚 Documentation

According to [ConstantLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ConstantLR.html#torch.optim.lr_scheduler.ConstantLR) "*Decays the learning rate of each parameter group by a small constant factor until the number of epoch reaches a pre-defined milestone: total_iters*"  i.e with `factor=0.5`, `lr=0.05`  and `total_iters=4` I would assume

```
#lr = 0.025 
#lr = 0.0125
#lr = 0.00625
#lr = 0.003125
#lr = 0.5  #Back to initial
```
i.e

```
lr[i+1] = lr[0]*start_factor**i
```
where the docs writes 

```
 # Assuming optimizer uses lr = 0.05 for all groups
# lr = 0.025   if epoch == 0
# lr = 0.025   if epoch == 1
# lr = 0.025   if epoch == 2
# lr = 0.025   if epoch == 3
# lr = 0.05    if epoch >= 4
```

In the  [LinearLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.LinearLR.html) the learning-rate examples

```
# Assuming optimizer uses lr = 0.05 for all groups
# lr = 0.025    if epoch == 0
# lr = 0.03125  if epoch == 1
# lr = 0.0375   if epoch == 2
# lr = 0.04375  if epoch == 3
# lr = 0.005    if epoch >= 4
 scheduler = LinearLR(self.opt, start_factor=0.5, total_iters=4)
```

increases after epoch 0 until epoch 4 where it drops again (why?). As far as I understand LinearLR we get the learning-rate in each step by

```
lr[i+1] = lr[i]*start_factor**(i)
```

Generally it would be great, if you provided a (short) formular of how you calculate each learning rate in the schedulers since the wording can be interpreted differently by different persons but formulars (often) does not.

### PR DESCRIPTION
**Pull request by zeshengzong**
Fixes #68058 

### Before

image

### After

image

### CODE REVIEW

**Comment by zeshengzong** [comment]
Hello @janeyx99 , please help review this one when available, thanks!

**Comment by janeyx99** [comment]
Why does this deviate so much from the first attempt:

image

**Comment by zeshengzong** [comment]
> Why does this deviate so much from the first attempt:
> 
> https://github.com/pytorch/pytorch/pull/71231/files#diff-036a7470d5307f13c9a6a51c3a65dd014f00ca02f476c545488cd856bea9bcf2
> 

Hi @janeyx99 , the previous version express the lr will change linearly with iteration, but not describe actual implementation and how to compute it.

image

https://github.com/pytorch/pytorch/blob/062387fb53651a7720a94a6fa65463821da9ce94/torch/optim/lr_scheduler.py#L735-L746

I think formula is to resolve the query about "how to computed it" without looking at code, otherwise seems unnecessary to have one. WDYT?  Thanks!
